model/Lite/Classes/LitePackage.md

```diff
@@ -0,0 +1,39 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# LitePackage  
+
+## Summary
+
+LitePackage contains the minimum part for package.
+
+## Description
+
+LitePackage contains the minimum part for package.
+
+## Metadata
+
+- name: LitePackage
+- SubclassOf: /Software/Package
+- Instantiability: Concrete
+
+## Properties
+
+- packageComment
```

Comment:
There already is a property `comment` on every subclass of `Element`, see [here](https://github.com/spdx/spdx-3-model/blob/cb37ac018d464cabc11d4b590c7830a489c8f1f5/model/Core/Classes/Element.md?plain=1#L39).